src/styles/application.css

```diff
@@ -0,0 +1,13 @@
+.sto-app {
+  display: flex;
+  margin: auto 1.7em;
+}
+
+.sto-app__icon {
```

Comment:
I didn't notice before but we used to prefer `--icon` to `__icon` for css modifier according to [BEM naming standard](http://getbem.com/naming/)

Reply:
According to BEM, --icon is a modifier of sto-app block. icon is an element of sto-app, not a modifier